### ISSUE TITLE
Add DCC integration

### ISSUE THREAD

**Issue by scott-wilson** [scott-wilson/checks#11]
This likely depends on the following issues:

- [ ] #9 
- [ ] #10 

But, we would want the UIs to be integrated in the following applications.

- [ ] Blender
- [ ] Gaffer
- [ ] Natron
- [ ] Houdini
- [ ] Katana
- [ ] Maya
- [ ] Nuke

The commercial DCCs would require licenses, so we may only be able to do this if we have valid licenses donated to us, or someone with valid licenses to develop the integrations. Also, we would need to figure out what are the versions that we will support (latest, latest + X releases, etc).

**Comment by aditiapratama** [comment]
Hi @scott-wilson this is wonderful tools. Anyway for example if I want to ship openchecks python package in blender addon, will it require to have rust as well?

**Comment by scott-wilson** [comment]
> Hi @scott-wilson this is wonderful framework. Anyway for example if I want to ship openchecks python package in blender addon, will it require to have rust as well?

Yup. However, I've made sure to publish compiled binaries for Linux, Windows, and macOS. So, all you should need to do is either pip install the library or vendor it with the rest of the code, and it should work as expected without having to be in Rust land.

**Comment by aditiapratama** [comment]
ah great then, I'll try tinkering with it this weekend and will let you know. Thanks
